src/Database/DBA.php

```diff
@@ -56,7 +56,7 @@ public static function getConnection()
 	}
 
 	/**
-	 * @brief Returns the MySQL server version string
```

Comment:
This doc block already had a long description making the `@brief` tag warranted.

Reply:
But the text below wasn't the long form of the `@brief` part.

Reply:
Fair enough.

Reply:
Where does `@brief` come from? At least PHPStorm doesn't know what to do with it. I found the "official" way of separating summary from description here: https://docs.phpdoc.org/latest/guides/docblocks.html#summary ... Do we really need this tag at all?

Reply:
@rabuzarus introduced it as a summary.